### PR TITLE
Packaging fix

### PR DESCRIPTION
**Pull request by kellertk**
*Issue #, if available:* #531

*Description of changes:* Overhauls the package action to pull a token from AWS and use that token to push changes back to master. The token that's pulled from AWS has permission to bypass branch protection rules.

---

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Comment by kellertk** [comment]
**Do not merge this to master as is**

This is using node 16 in action.yml right now. This needs to be modified to target the right node version before merging.

**Review by peterwoodworth — APPROVED** [review]

**Review by peterwoodworth — APPROVED** [review]

**Pull request merged**
